expyfun/_experiment_controller.py

```diff
@@ -840,13 +852,14 @@ def flip(self, when=None):
         self._win.switch_to()
         gl.glFinish()
         self._win.flip()
-        # this waits until everything is called, including last draw
-        gl.glClear(gl.GL_COLOR_BUFFER_BIT)
-        gl.glBegin(gl.GL_POINTS)
-        gl.glColor4f(0, 0, 0, 0)
-        gl.glVertex2i(10, 10)
-        gl.glEnd()
-        gl.glFinish()
+        if not self._enable_video:
+            # this waits until everything is called, including last draw
+            gl.glClear(gl.GL_COLOR_BUFFER_BIT)
+            gl.glBegin(gl.GL_POINTS)
+            gl.glColor4f(0, 0, 0, 0)
+            gl.glVertex2i(10, 10)
+            gl.glEnd()
+            gl.glFinish()
```

Comment:
Did you need to uncomment all lines? Maybe `glEnd` and `glFinish` can stay unindented?